client/my-sites/theme/index.web.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { makeLayout } from 'controller';
+import { details, fetchThemeDetailsData } from './controller';
+import { siteSelection } from 'my-sites/controller';
+
+export default function( router ) {
+	if ( config.isEnabled( 'manage/themes/details' ) ) {
+		router( '/theme/:slug/:section?/:site_id?', siteSelection, fetchThemeDetailsData, details, makeLayout );
```

Comment:
You could remove `fetchThemeDetailsData` here, since it has a check that skips any actions on the client side anyway, since data is gathered by the data component's lifecycle methods instead. Or leave it in, to be closer to isomorphism with `index.node.js`.